caffe2/ideep/utils/ideep_context.h

```diff
@@ -8,7 +8,15 @@
 
 namespace caffe2 {
 
-BaseStaticContext* GetIDEEPStaticContext();
+#ifdef _WIN32
+#define CAFFE2_IDEEP_EXPORT __declspec(dllexport)
```

Comment:
Why do we need specialization like this instead of just using `CAFFE2_EXPORT`?

Reply:
Not aware of this macro. Will fix it soon.